test/trento/clusters_test.exs

```diff
@@ -278,7 +286,90 @@ defmodule Trento.ClustersTest do
         assert message.env == %{
                  "provider" => %{kind: {:string_value, Atom.to_string(provider)}},
                  "cluster_type" => %{kind: {:string_value, Atom.to_string(cluster_type)}},
-                 "filesystem_type" => %{kind: {:string_value, Atom.to_string(:mixed_fs_types)}}
+                 "filesystem_type" => %{kind: {:string_value, Atom.to_string(:mixed_fs_types)}},
+                 "ensa_version" => %{
+                   kind: {:string_value, Atom.to_string(EnsaVersion.mixed_versions())}
+                 }
+               }
+
+        assert message.target_type == "cluster"
+
+        :ok
+      end)
+
+      assert :ok = Clusters.request_checks_execution(cluster_id)
+    end
+
+    test "should start a checks execution on demand for ascs_ers clusters with ENSA 1 version" do
```

Comment:
I haven't included a test(s) for the case where a cluster has only one SAP system, but that seems low-value, what do you think?

Reply:
Agreed